api/envoy/extensions/load_balancing_policies/least_request/v3/least_request.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package envoy.extensions.load_balancing_policies.least_request.v3;
+
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.load_balancing_policies.least_request.v3";
+option java_outer_classname = "LeastRequestProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/load_balancing_policies/least_request/v3;least_requestv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Least Request Load Balancing Policy]
+
+// This configuration allows the built-in LEAST_REQUEST LB policy to be configured via the LB policy
+// extension point. See the :ref:`load balancing architecture overview
+// <arch_overview_load_balancing_types>` for more information.
+// [#extension: envoy.clusters.lb_policy]
+message LeastRequest {
```

Comment:
This should also have the [`active_request_bias`](https://github.com/envoyproxy/envoy/blob/f2b3fb39955414720718fdb8ccb4e0fb815523bf/api/envoy/config/cluster/v3/cluster.proto#L420) and [`slow_start_config`](https://github.com/envoyproxy/envoy/blob/f2b3fb39955414720718fdb8ccb4e0fb815523bf/api/envoy/config/cluster/v3/cluster.proto#L424) fields.

Reply:
I don't know how I missed those. Done.